tests/DifferTest.php

```diff
@@ -273,42 +273,7 @@ public function textProvider(): array
                 'b',
             ],
             [
-                "--- Original\n+++ New\n@@ @@\n-ba\n+bc\n",
-                'ba',
-                'bc',
-            ],
-            [
-                "--- Original\n+++ New\n@@ @@\n-ab\n+cb\n",
-                'ab',
-                'cb',
-            ],
-            [
-                "--- Original\n+++ New\n@@ @@\n-abc\n+adc\n",
-                'abc',
-                'adc',
-            ],
-            [
-                "--- Original\n+++ New\n@@ @@\n-ab\n+abc\n",
-                'ab',
-                'abc',
-            ],
-            [
-                "--- Original\n+++ New\n@@ @@\n-bc\n+abc\n",
-                'bc',
-                'abc',
-            ],
-            [
-                "--- Original\n+++ New\n@@ @@\n-abc\n+abbc\n",
-                'abc',
-                'abbc',
-            ],
-            [
-                "--- Original\n+++ New\n@@ @@\n-abcdde\n+abcde\n",
-                'abcdde',
-                'abcde',
-            ],
-            [
```

Comment:
why removing those cases?

Reply:
These are all repetition of the same type of case;
i.e. test diff generation of one line vs. another line.
For this `diff` package that is not of much interest as the output is line based.

I wrote those tests with having column diff as additional option in mind for later. However I never made that and when re-reviewing this I figured it would not make sense adding it to the output builder where these test belong to, so I removed the duplication.